Makefile

```diff
@@ -1264,6 +1268,10 @@ test_integrations_frankenphp: global_test_run_dependencies
 	$(call run_tests_debug,--testsuite=frankenphp-test)
 test_integrations_roadrunner: global_test_run_dependencies tests/Frameworks/Roadrunner/Version_2/composer.lock-php$(PHP_MAJOR_MINOR)
 	$(call run_tests_debug,tests/Integrations/Roadrunner/V2)
+setup_grpc:
+	if [ ! -f /opt/php/debug/php.ini ] ; then sudo pecl install grpc ; echo "extension=grpc.so" > /opt/php/debug/php.ini ;fi
```

Comment:
This takes too much time. We'll have to add it to the dockerfiles themselves. Let me try doing that.

Reply:
grpc is now present in the 8.0 to 8.4 buster containers which are used to run these tests - you may remove this now :-)

Reply:
Thank you!

Reply:
I removed the code of setup_grpc.